compiler/env/OMRClassEnv.hpp

```diff
@@ -111,6 +111,9 @@ class OMR_EXTENSIBLE ClassEnv
    char *classSignature_DEPRECATED(TR::Compilation *comp, TR_OpaqueClassBlock * clazz, int32_t & length, TR_Memory *) { return NULL; }
    char *classSignature(TR::Compilation *comp, TR_OpaqueClassBlock * clazz, TR_Memory *) { return NULL; }
 
+   // Get the virtual function table entry at a specific offset from the class
```

Comment:
Please provide a Doxygen comment for the API (i.e., the parameters and return value as well).